src/models/index.ts

```diff
@@ -44,7 +44,6 @@ SubscriptionModel.init({
 	gitHubInstallationId: DataTypes.INTEGER,
 	jiraHost: DataTypes.STRING,
 	selectedRepositories: DataTypes.ARRAY(DataTypes.INTEGER),
-	repoSyncState: DataTypes.JSONB,
```

Comment:
So after we remove this the repoSyncState column won't be used anymore, correct?

Reply:
That's correct :)  I have another ticket to remove it altogether from the DB after this is merged.